src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-mon calendar--month-length-31">
+      <div class="calendar__field">
```

Comment:
Use a loop (e.g. for, while, or do-while loop) to create the 'calendar__day' divs instead of repeating the same line of code multiple times.